spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -7,10 +7,10 @@ This specification defines a powerful set of complementary services that help to
 * A sophisticated, typesafe _dependency injection_ mechanism, including the ability to select dependencies at either development or deployment time, without verbose configuration
```

Comment:
I suggest to define cdi full and cdi liter concept first in this chapter and then refer to it.